src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -508,18 +573,34 @@ export const generateContent: TranslationFn = content => {
   const displayState = currentStateFn(userCase.state).at(
     (userCase.state === State.OfflineDocumentReceived ? userCase.previousState : userCase.state) as State
   );
+  const latestRequestForInformation = userCase.requestsForInformation?.at(0)?.value;
+  const isSoleRequestForInformation =
+    latestRequestForInformation !== null && latestRequestForInformation?.requestForInformationSoleParties !== undefined;
+  const soleParties = isSoleRequestForInformation
+    ? latestRequestForInformation.requestForInformationSoleParties
+    : undefined;
+  const isSoleRequestForInformationForApplicant = soleParties === 'applicant';
```

Comment:
We may be able to simplify this by relying on the optional chaining:
```suggestion
  const soleParties = latestRequestForInformation?.requestForInformationSoleParties;
  const isSoleRequestForInformationForApplicant = soleParties === 'applicant';
```

Reply:
Done:

`const isApplicantAbleToRespondToRequestForInformation =
    latestRequestForInformation?.requestForInformationSoleParties === 'applicant';`